src/pfs_target_uploader/utils/ppp.py

```diff
@@ -489,7 +489,7 @@ def cobraMoveCost(dist):
         """optional: penalize assignments where the cobra has to move far out"""
         return 0.1 * dist
 
-    def netflowRun_single(Tel, sample):
+    def netflowRun_single(Tel, sample, otime = "2024-05-20T08:00:00Z"):
```

Comment:
@wanqqq31 I'm wondering if this is an obsolete change or not. Could you please confirm it?